src/Microsoft.AspNet.Localization/ApplicationBuilderExtensions.cs

```diff
@@ -63,12 +33,7 @@ public static class ApplicationBuilderExtensions
                 throw new ArgumentNullException(nameof(options));
             }
 
-            if (defaultRequestCulture == null)
```

Comment:
It's initialized directly via the backing field https://github.com/aspnet/Localization/pull/170/files#diff-61f74221a354a19742c79b0c32c46423R15